src/partials/contacts.html

```diff
@@ -0,0 +1,80 @@
+      <main>
```

Comment:
Wyrzuć tylko main bo jest to część HTML która jest już w indexie i będzie gotowe. W footer tak samo ;)